src/components/Search/SearchResultsListItem.tsx

```diff
@@ -68,6 +68,21 @@ function SearchResultsListItem({
   const commitMessage = getLatestCommitMessage(item);
   const itemDate = new Date(item.push_timestamp * 1000);
 
+  const localDateTime = itemDate
+    .toLocaleString(undefined, {
+      year: '2-digit',
+      month: '2-digit',
+      day: '2-digit',
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+      timeZoneName: 'short',
+    })
+    .replace(',', '') // Removes the comma between the date and time
+    .replace(/(\d{2})\/(\d{2})\/(\d{2})/, '$2/$1/$3'); // Reformat date as MM/DD/YY
+
+  const timeZone = localDateTime.split(' ').pop();
+
```

Comment:
You need to show the utc time in the tooltip not the localtime.

Reply:
Ahhh Ok 
That makes more sense... will correct that now

Reply:
Oh I missed the part you asked why I wasn't using dayjs. Well cause you would need to install additional plugins to work with timezones, so I thought the native javascript way would be better